issuing/cards.go

```diff
@@ -0,0 +1,94 @@
+package issuing
+
+import (
+	"time"
+
+	"github.com/checkout/checkout-sdk-go/common"
+)
+
+type CardType string
+
+const (
+	Physical CardType = "physical"
+	Virtual  CardType = "virtual"
+)
+
+type CardStatus string
+
+const (
+	CardActive    CardStatus = "active"
+	CardInactive  CardStatus = "inactive"
+	CardRevoked   CardStatus = "revoked"
+	CardSuspended CardStatus = "suspended"
+)
+
+type (
+	CardTypeResponse interface {
+		GetType() CardType
+		GetDetails() interface{}
+	}
+
+	PhysicalCardType struct {
```

Comment:
This structs should be declared in lower case, so that they are "private"

Reply:
👍🏽